src/notional/core.py

```diff
@@ -84,28 +81,36 @@ def __getitem__(self, params):
 
         compose_func = self.__compose__
 
-        if type(params) is tuple:
+        # __getitem__ only accepts a single parameter...  if the caller provides
+        # multiple params, they will be converted and passed as a tuple.  this method
+        # also accepts a list for readability when composing from ORM properties
+
+        if params and type(params) in (list, tuple):
             return compose_func(*params)
 
         return compose_func(params)
 
 
 class GenericObject(BaseModel, metaclass=ComposableObjectMeta):
-    """The base for all API objects."""
+    """The base for all API objects.
+
+    As a general convention, data fields in lower case are defined by the
+    Notion API.  Properties in Title Case are provided for convenience.
+    """
 
     def __setattr__(self, name, value):
         """Set the attribute of this object to a given value.
 
-        The implementation of `BaseModel.__setattr__` does not allow for properties.
+        The implementation of `BaseModel.__setattr__` does not support property setters.
 
         See https://github.com/samuelcolvin/pydantic/issues/1577
         """
         try:
             super().__setattr__(name, value)
         except ValueError as err:
             setters = inspect.getmembers(
-                self.__class__,
-                predicate=lambda x: isinstance(x, property) and x.fset is not None,
```

Comment:
It might be better to keep this check...  At least, removing it seems unnecessary.